DEPLOYMENT.md

```diff
@@ -22,22 +22,30 @@ On this page we list the various steps needed for deployments. The first section
 
     EOT
     ```
-  * Install / setup postgress
+  * Install / setup postgreSQL 15
     ```
     yum install centos-release-scl -y
     yum install https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
-    yum install postgresql11-server postgresql11-devel -y
-    export PATH=$PATH:/usr/pgsql-11/bin
+    yum install postgresql15 postgresql15-server postgresql15-devel -y
 
-    /usr/pgsql-11/bin/postgresql-11-setup initdb
-    systemctl enable postgresql-11
-    systemctl start postgresql-11
+    cat >/etc/profile.d/pg_conf.sh <<EOF
```

Comment:
you sure you don't need to make `pg_config.sh` executable with `chmod +x pg_config.sh`?